src/main/java/com/google/solutions/ml/api/vision/BatchRequestsTransform.java

```diff
@@ -66,7 +67,8 @@ public Integer apply(String input) {
                       return random.nextInt(getKeyRange());
                     }
                   }))
-          .apply("Group Into Batches", GroupIntoBatches.ofSize(getBatchSize()))
+          .apply("Group Into Batches", GroupIntoBatches.<Integer, String>ofSize(getBatchSize())
+              .withMaxBufferingDuration(Duration.standardSeconds(30)))
```

Comment:
30sec looks like a magic number -- should it be configurable via pipeline args?

Reply:
Made it configurable.